src/contexts/PredictoorsContext.tsx

```diff
@@ -377,6 +375,7 @@ export const PredictoorsProvider: React.FC<TPredictoorsContextProps> = ({
           return
 
         lastCheckedEpoch.current = newCurrentEpoch
+        lastTimeframe = timeFrameInterval
```

Comment:
need to use the "current" property of the mutable object type for assigning value